app/javascript/blockchain/KeyTools.js

```diff
@@ -18,6 +18,16 @@ class KeyTools {
     return this.web3.eth.accounts.wallet[0].address;
   }
 
+  /** Generates random valid address */
+  generateRandomAddress(){
```

Comment:
Should generated address be tied to user's mnemonic? 
In this case user could have a private key for new address

Reply:
I think we can remove it now.